libs/ngrx-toolkit/src/lib/devtools/internal/devtools-syncer.service.ts

```diff
@@ -54,21 +54,47 @@ export class DevtoolsSyncer implements OnDestroy {
   #currentId = 1;
 
   readonly #connection: Connection = this.#isBrowser
-    ? window.__REDUX_DEVTOOLS_EXTENSION__
-      ? window.__REDUX_DEVTOOLS_EXTENSION__.connect(this.#devtoolsConfig)
-      : dummyConnection
+    ? this.#initDevtoolsConnection()
     : dummyConnection;
 
   constructor() {
     if (!this.#isBrowser) {
-      return;
+      console.warn(
+        '[DevtoolsSyncer] Not running in browser. DevTools disabled.',
+      );
     }
   }
 
   ngOnDestroy(): void {
     currentActionNames.clear();
   }
 
+  #initDevtoolsConnection(): Connection {
+    const extension = window.__REDUX_DEVTOOLS_EXTENSION__;
+    if (!extension) {
+      console.warn('[DevtoolsSyncer] Redux DevTools extension not found.');
```

Comment:
i think it can be annoying to throw warnings if DevTools are not
there. Especially since they can also be enabled in production.